src/Main.java

```diff
@@ -1,5 +1,10 @@
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        let dog = 8.0
+        let cat = 3.6
+        let paper = 763789
```

Comment:
```suggestion
        var dog = 8.0;
        var cat = 3.6;
        var paper = 763789;
```
в Java нет let и также нужно всегда ставить ; в конце)